Packager/ProjectSettings/ClusterInputManager.asset

```diff
@@ -0,0 +1,6 @@
+%YAML 1.1
```

Comment:
Looks like you've updated the project or something and now it's adding all kinds of stuff. Can we get this change in without changing any project versions?